### ISSUE TITLE
ionic-angular templates problem

### ISSUE THREAD

**Issue by AdoBeatTheWorld** [driftyco/ionic#11339]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ x] **2.x**
[ ] **3.x**

**I'm submitting a ...**  (check one with "x")
[ x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

start a ionic pro with : ionic start TestPro --v2
then generate a page with: ionic -g page Test
there is an error in the generated page.
IonicModule.forChild($CLASSNAME),
should be :
IonicModule.forRoot($CLASSNAME),

**Expected behavior:**

Error.

**Steps to reproduce:**

**Related code:**

*ionic-angular\templates\page\module.ts.tmpl* line 9 :
```
imports: [
    IonicModule.forChild($CLASSNAME),
  ],
```
should be :
```
imports: [
    IonicModule.forRoot($CLASSNAME),
  ],
```

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Cordova CLI:6.5.0
Ionic CLI Version:2.2.2
Ionic App Lib Version:2.2.1
os:Windows 7
Node Version:v6.9.4
```
ionic -v:
```
2.2.2
```

**Comment by AdoBeatTheWorld** [comment]
I checked ionic-angular@3.0.1, there is no IonicModule.forChild() method, only the IonicModule.forRoot()

**Comment by AdoBeatTheWorld** [comment]
My collegue told me this must be 
```
IonicPageModule.forChild()
```

**Comment by MTechDE** [comment]
Already fixed and merged with upcoming release 3.1

You can close that issue thanks.

**Comment by AdoBeatTheWorld** [comment]
thanks
